### PR TITLE
UI Dev 4: End of January Release w/Benchmarking

### PR DESCRIPTION
**Pull request by Bianca-Morris**
### Changelog:
- UIs for Benchmarking Data
- Google Analytics implementation
- Navigation edits to accommodate new documentation, about, data pages
- Various other front-end tweaks to home page, user pages, etc

#### Merged Features:
- [x] https://github.com/smaht-dac/smaht-portal/pull/52   _- Bianca_
       - Contains changes from [`bm-benchmarking-tables`](https://github.com/smaht-dac/smaht-portal/tree/bm-benchmarking-tables); basic benchmarking ui layout for static sections
       - Also contains changes from [`uo-embedded-table-fix`](https://github.com/smaht-dac/smaht-portal/tree/uo-embedded-table-fix); related to [this ticket](https://trello.com/c/aYjvxsh3/844-embedded-search-table-scroll-fix)
- [x] Google Analytics Implementation ([Trello Ticket](https://trello.com/c/6xMuj1OV/850-google-analytics-for-smaht-implementation), branch: `ga4`) _- Utku_
- [x] Top nav edits for data/benchmarking https://github.com/smaht-dac/smaht-portal/pull/62 (Branch: [`bm-nav-edits`](https://github.com/smaht-dac/smaht-portal/pull/62)) _- Bianca_
- [x] MWF + Item Page Fixes for Alex V., Doug, and Michele ([Trello Ticket - MWF object value](https://trello.com/c/tDD77iVi/847-bugfix-for-metaworkflows), [Trello Ticket - QC Value Bug](https://trello.com/c/lTO9fNBI/868-qc-value-bug), [Trello Ticket - Truncated Fields](https://trello.com/c/73n6sPGX/842-truncated-fields-in-smaht-fix), Branch: [`cfm_qc_fix`](https://github.com/4dn-dcic/shared-portal-components/compare/cfm-qc_fix)) **Released as** [SPC v0.1.73](https://github.com/4dn-dcic/shared-portal-components/releases/tag/0.1.73), pending final review & merge _- Cesar_
- [x] Facet header fixes/improvements ([Trello Ticket](https://trello.com/c/ja9rsety/855-adjust-facetheader-and-fontawesomeiconv6-components-to-make-them-a-bit-more-modular-usable-with-different-font-border-and-icon-c), Branches: [Smaht-portal: `cfm-facet_styles`](https://github.com/smaht-dac/smaht-portal/pull/54), [SPC: `cfm-facet_styles`](https://github.com/4dn-dcic/shared-portal-components/compare/cfm-facet_styles)) **Released as** [SPC v0.1.73](https://github.com/4dn-dcic/shared-portal-components/releases/tag/0.1.73), pending final review & merge  _- Cesar_
- [x] UI Edits for Benchmarking (searchHrefs, additional facet styling, side nav edits) (Branch: `bm-benchmarking-styling`, PR: https://github.com/smaht-dac/smaht-portal/pull/64, SPC Branch: `bm-benchmarking-table-fix`, SPC PR: https://github.com/4dn-dcic/shared-portal-components/pull/244)_- Bianca_
- [x] Copy edits for About Pages _ (Branch: https://github.com/smaht-dac/smaht-portal/pull/61) - Elizabeth & Cesar
- [x] Streamline layouts of RST static pages ([Trello Ticket](https://trello.com/c/HXmoKRmX/852-static-page-adjustments-get-rst-md-jsx-styling-markup-to-match)) _- Cesar & Utku_

#### Ongoing Work:
- [x] Schema Edits for Benchmarking table columns & initial facets ([Trello Ticket](https://trello.com/c/JM9EPr6f/425-smaht-benchmarking-columns), Branch: [`drr_file_ui`](https://github.com/smaht-dac/smaht-portal/tree/drr_file_ui)) _- Doug_
- [x] Schema/Data Model Edits for Benchmarking searchHrefs (Branch: [`drr_file_links`](https://github.com/smaht-dac/smaht-portal/tree/drr_file_ui)) _- Doug_
- [x] Remaining style tweaks _- Bianca, Shannon, Cesar_

#### Additional Todos:

- [ ] Additional Doc pages (?)
- [ ] _??? Does anything else remain for MVP ???_
- [x] Larger discussion re: which fields to include facets for _(Most Relevant to: Doug, Elizabeth, Shannon, Bianca, Cesar)_
   - Decisions made: https://docs.google.com/document/d/1V1np3ZPf91SB6RC6-C8pi9vKzI2DRdU8ha8oMMQkuvI/edit
- [x] Adjustments to Directory/Grid pages and/or Navbar dropdown to hide said pages (needs more discussion/confirmation w/Elizabeth) ([Trello Ticket](https://trello.com/c/s2Ls1FdV/857-directory-page-edits))

### CODE REVIEW

File: src/encoded/types/tracking_item.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from typing import Any, Dict, Optional, List
```
Comment (drio18) [inline src/encoded/types/tracking_item.py:1]: Will need to add an insert for this item to `workbook-inserts` for tests to pass.

**Review by drio18 — COMMENTED** [review]
Mostly looks good to me. Suggested some changes to pass tests for the addition of the new item type (TrackingItem) as well as a couple minor things.

File: src/encoded/schemas/tracking_item.json [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+                                        "type" : "integer",
+                                        "title" : "Clicks from Browse",
+                                        "description": "Number of times this ExperimentSet has been clicked on from the Browse page."
+                                    },
+                                    "ga:productListViews" : {
+                                        "type" : "integer",
+                                        "title" : "List Views",
+                                        "description": "Number of times this ExperimentSet has been seen present in a Browse or Search results page."
+                                    },
+                                    "ga:metric2" : {
```
[the remark this answers is not in the document]
Comment (utku-ozturk) [reply src/encoded/schemas/tracking_item.json:437]: @drio18 `ga:` prefixed fields are built-in property names defined in Google Analytics Property. We might rename them while converting into TrackingItem but it absolutely makes future updates harder.

File: src/encoded/schemas/tracking_item.json [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+                                        "type" : "integer",
+                                        "title" : "Downloads"
+                                    },
+                                    "ga:metric1" : {
+                                        "type" : "integer",
+                                        "title" : "Cumalative Filesize"
+                                    },
+                                    "ga:calcMetric_PercentRangeQueries" : {
```
[the remark this answers is not in the document]
Comment (utku-ozturk) [reply src/encoded/schemas/tracking_item.json:445]: @drio18 https://github.com/smaht-dac/smaht-portal/pull/45#discussion_r1482049253

File: src/encoded/types/tracking_item.py (hunk 1 above)
Comment (utku-ozturk) [reply src/encoded/types/tracking_item.py:1]: @drio18 done: https://github.com/smaht-dac/smaht-portal/blob/c5e6a52a3f61b4bfe6c8af258879ee08cd575f2c/src/encoded/tests/data/workbook-inserts/tracking_item.json

**Review by utku-ozturk — APPROVED** [review]
looks good to me!

File: src/encoded/types/tracking_item.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+from snovault import collection, load_schema, calculated_property, Item as SnovaultItem
+from encoded_core.types.tracking_item import TrackingItem as CoreTrackingItem
+
+# from .acl import ONLY_ADMIN_VIEW_ACL, ALLOW_OWNER_EDIT_ACL
+from .base import Item
+
+@collection(
+    name='tracking-items',
```
[the remark this answers is not in the document]
Comment (drio18) [reply src/encoded/types/tracking_item.py:10]: I believe so. The name is a little misleading as the view permissions are also status-dependent (see [here](https://github.com/smaht-dac/smaht-portal/blob/main/src/encoded/types/acl.py#L29-L39)), but that ACL should prevent non-admins from creating/editing the items.

File: src/encoded/types/tracking_item.py [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    properties={
+        "title": "TrackingItem",
+        "description": "For internal tracking of Fourfront events",
+    })
+class TrackingItem(Item, CoreTrackingItem):
+    item_type = 'tracking_item'
+    schema = load_schema("encoded:schemas/tracking_item.json")
+    embedded_list = []
+
+    def _update(self, properties: Dict[str, Any], sheets: Optional[List] = None) -> None:
```
[the remark this answers is not in the document]
Comment (Bianca-Morris) [reply src/encoded/types/tracking_item.py:20]: @willronchetti a heads up - you may have to resolve this in your branch.

**Pull request merged**